app/src/main/java/com/example/hsh/homesweethome/Models/Furniture.java

```diff
@@ -34,6 +34,9 @@
     @SerializedName("furnitureBrandImageUrl")
     @Expose
     private String furnitureBrandImageUrl;
+    @SerializedName("inStore")
```

Comment:
What does this do again bro? PS for asking i just wanna learn

Reply:
This serializes whatever coming in to the app so that it can understand